### ISSUE TITLE
spongebobcase.spongebobsay() prints two copies

### ISSUE THREAD

**Issue by jameslamb** [jayqi/spongebob#29]
I installed `spongebobcase` 0.4.0 from PyPi today and ran this

```python
import spongebobcase
print(spongebobcase.spongebobsay("this is working"))
``` 

And got two spongebobs!

![image](https://user-images.githubusercontent.com/7608904/88321031-2430a900-cce4-11ea-8db8-3a08c3c981dc.png)

**Comment by jameslamb** [comment]
oh I see, I think you must be both printing AND returning. When I assign to a variable I only get one printed

![image](https://user-images.githubusercontent.com/7608904/88324585-84761980-cce9-11ea-97f9-67b77ecaa9a4.png)

**Comment by jayqi** [comment]
There is a `print_` argument that controls whether it prints internally in the function. Happy to take feedback about if that seems intuitive. 

https://jayqi.github.io/spongebob/py-pkg/api-reference/spongebobsay/#spongebobcase.spongebobsay.spongebobsay

It's the same way the R function works

https://jayqi.github.io/spongebob/r-pkg/reference/spongebobsay.html

R has the concept of returning invisibly but unfortunately Python does not.

**Comment by jameslamb** [comment]
ooooo I didn't realize! Ok my fault for not RTFM.

My expectation from using [`{cowsay}`](https://cran.r-project.org/web/packages/cowsay/index.html) in the past was that I could just run this function and it would print one thing, without me needing to do any assignment.

I think that argument does what I want, but seeing it I would have expected it's default value to be `False`. I expect that the interactive case is a lot more common than the "return a value" case.

Anyone, that did answer my question so you can close this, thanks!

**Comment by jayqi** [comment]
Yeah, I mean, it's kind of tricky. I meant for the `print_` argument to be there for the interactive case, and I personally do `_ = spongebobsay("whatever")` to suppress the returned string. In R, it both prints and returns the string the same way, but the string is just returned invisibly.

**Comment by jameslamb** [comment]
Re-reading this conversation from 4+ years ago, it looks like this was intentional behavior. This definitely can be closed.
